src/components/socialLogin/tests/Google.test.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+import GoogleButton from '../GoogleButton';
+import SignUpPage from '../../../pages/Signup/index';
+
+describe('Google Sign in Button', () => {
+  it('should render without crashing', () => {
+    const wrapper = renderer.create(<GoogleButton />);
+    expect(wrapper.toJSON()).toMatchSnapshot();
+  });
+
+  it('signup page mount', () => {
+    const wrapper = shallow(<SignUpPage />);
```

Comment:
Use `mount` instead

Reply:
@adesege unfortunately, when using mount the test fails